src/style.css

```diff
@@ -1 +1,64 @@
 /* Styles go here */
+
+html,
+body {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  height: 100%;
```

Comment:
height in percent works only if the parent element has the exact height set
But the header does not have it as I see